lib/spreadbase/codecs/open_document_12.rb

```diff
@@ -1,4 +1,5 @@
 require 'zip'
+require 'zip/version'
```

Comment:
With the new check, this is not required anymore :slightly_smiling_face:

Reply:
I've removed it